src/main/java/crazypants/enderio/conduit/liquid/EnderLiquidConduit.java

```diff
@@ -40,6 +41,12 @@
   public static final String ICON_INSERT_KEY = "enderio:liquidConduitAdvancedOutput";
   public static final String ICON_IN_OUT_KEY = "enderio:liquidConduitAdvancedInOut";
 
+  public static final String ICON_KEY_INPUT = "enderio:itemConduitInput";
```

Comment:
How about initialize using those constants =in ItemConduit? e.g.
```java
  public static final String ICON_KEY_INPUT = ItemConduit.ICON_KEY_INPUT;
```
or maybe just reference the constants in call site.

Reply:
The idea crossed my mind, but I figured we might want to make custom icons with some outline or something to make them slightly easier to tell apart. The idea isn't really that they are supposed to use the same icon, it just turned out to work well enough that I didn't bother making a custom version for the fluids. On the other hand, it does work alright with using the same, so I guess I could reference the existing ones.

Reply:
I'm actually ok with any version, but I'd prefer to use my suggested version. It makes clears that this class is intentionally reusing the item conduit version of icons while making it clear this is not a typo.